docs/dev/development-process/development-process.rst

```diff
@@ -0,0 +1,203 @@
+*******************
+Development Process
+*******************
+
+.. contents:: Content of this document
+    :local:
+    :depth: 2
+
+
+The Artemis Development Process Model
+===========================================
+
+.. figure:: ./artemis-process-model.png
+    :align: center
+    :alt: Artemis Process Model - Activity Diagram
+    :figclass: align-center
+
+1. Submit a Feature Request
+---------------------------
+The initial step in our development process involves the creation of a feature request, which is accomplished through the submission of a GitHub Issue. 
+This action can be performed by any stakeholder, including developers, users, or maintainers. 
+The feature request should include a detailed description of the desired functionality, as well as any relevant information that may be useful to the development team. 
+This information may include the rationale for the feature, the expected benefits, and any potential risks or challenges that may be associated with the implementation of the feature.
+
+2. Evaluate Feature Request
+---------------------------
+Once a feature request has been submitted, the maintainers will evaluate the request together with the development team to determine its feasibility and potential impact on the system.
+
+3. Formulate a Feature Proposal
+-------------------------------
+If the feature request is deemed feasible, the development team will create a feature proposal that extensively describes the proposed feature. This step will be performed by the development team and will consist of the artifacts mentioned in the model "Create Feature Proposal" below:
+
+.. figure:: ./aremtis-feature-proposal-flow.png
+    :align: center
+    :alt: Create a Feature Proposal - Activity Diagram
+    :figclass: align-center
+
+
+Step 1: Amend Feature Proposal Template to Feature Request on GitHub
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The Feature Proposal Template outlines the structure of the feature proposal and provides a guideline for the development team to follow.
+
+We have written a Github Action that automatically attaches the feature proposal template to the issue description of an issue when someone is assigned.
+As a developer who is responsible for the implementation of a feature request, you do not have to do any than register as an assignee in the issue. 
+
+.. note::
+      The GitHub Action will be skipped for issues that are labeled with "bug".
+
+.. literalinclude:: ../../../.github/ISSUE_TEMPLATE/feature-proposal--developer-.md
+    :caption: Artemis/.github/ISSUE_TEMPLATE/feature-proposal--developer-.md
+    :language: markdown
+
+
+Step 2: Requirements Engineering
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This section of the feature proposal includes a description of all Functional and Non-Functional Requirements that the feature should fulfill.
```

Comment:
> Functional and Non-Functional Requirements

This doesn't need to be capitalized, they are important concepts but not proper nouns